src/derived/slot.rs

```diff
@@ -696,16 +720,9 @@ where
         );
 
         if self.verified_at == revision_now {
-            let is_constant = match self.inputs {
-                MemoInputs::Constant => true,
-                _ => false,
-            };
-
             return Some(StampedValue {
```

Comment:
Let's inline `probe_memoized_value` into the call site? It doesn't do much, and we can put the condition into a match arm guard.